capture-sdk/screen-api-example-app/build.gradle.kts

```diff
@@ -113,4 +118,7 @@ dependencies {
     androidTestImplementation(libs.androidx.test.junit)
 }
 
+kapt {
+    correctErrorTypes = true
```

Comment:
[Request] Please add a short comment why this option is needed. Our future selves will thank us. 🙂